src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitManager.java

```diff
@@ -62,15 +62,39 @@ public ConsensusCommit start(String txId) {
 
   @Override
   public synchronized ConsensusCommit start(Isolation isolation) {
-    String txId = UUID.randomUUID().toString();
-    return start(txId, isolation);
+    return start(isolation, SerializableStrategy.EXTRA_WRITE);
```

Comment:
After some benchmarking and verification, we can change the default strategy based on the results.